tests/test_integration.py

```diff
@@ -0,0 +1,39 @@
+#    Copyright 2020 D-Wave Systems Inc.
+
+#    Licensed under the Apache License, Version 2.0 (the "License")
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+
+#        http: // www.apache.org/licenses/LICENSE-2.0
+
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+from subprocess import Popen, PIPE,STDOUT
+import unittest
+import os
+
+class IntegrationTests(unittest.TestCase):
+
+    def test_factoring(self):
+        cwd = os.getcwd()
+        p = Popen(["python", cwd+"/demo.py"], stdout=PIPE, stdin=PIPE, stderr=STDOUT)    
+        p.stdin.write(b'49\n')
+        output = p.communicate()[0]
+        output = str(output).upper()
+        print("Example output \n"+ output)
```

Comment:
This print adds a lot of noise to test output. Why do we need it?

Reply:
I need this in jenkins pipeline , as dont want to completely rely on automated tests
helpful for manual verification whenever needed ,
if there is better way to get output executed let me konw thanks

Reply:
In that case I would recommend a conditional debug output. Either via environment variable + `logging` or environment variable + `print`. Something like:
```
def debug_print(*args, **kwargs):
    if os.getenv('DEBUG_OUTPUT'):
        print(*args, **kwargs)

...

def test_x():
    ...
    debug_print("Example output\n" + output)
```

I would prefer `logging` approach more, but maybe the above is simpler.

FWIW, I'm working on `dwave-common` package that will standardize Ocean logging primitives like this.

Reply:
Added env + print for now will update the code once your common package is available that is the standard way

Reply:
Looks good, thanks!